crates/engine/util/src/reorg.rs

```diff
@@ -380,11 +380,8 @@ where
     let hashed_state = state_provider.hashed_post_state(outcome.state());
 
     let (blob_gas_used, excess_blob_gas) =
-        if chain_spec.is_cancun_active_at_timestamp(reorg_target.timestamp) {
-            (
-                Some(sum_blob_gas_used),
-                reorg_target_parent.next_block_excess_blob_gas(BlobParams::cancun()),
-            )
+        if let Some(blob_params) = chain_spec.blob_params_at_timestamp(reorg_target.timestamp) {
+            (Some(sum_blob_gas_used), reorg_target_parent.next_block_excess_blob_gas(blob_params))
```

Comment:
we could get rid of the if let some via .maybe_ call + map but debatable what's better here